src/components/ColorPicker.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+
+<div class="color-picker-overlay" @click="closeColorPicker">
+  <div class="container" :style="containerPosition" ref="container" @click.stop>
+  <div @click="saveColor" class="button save">&check;</div>
+  <v-color-picker 
+  class="myColorPicker"
+  v-on:input = "colorPickerColor"
+  dot-size = "40"
+  mode="hexa"
+  v-bind:dark= "true"
+  v-bind:value = "oldColor"
+  width="250"
+  canvas-height="100"
+></v-color-picker>
+<div @click="closeColorPicker" class="button close">&#9932;</div>
+  <div class="color-picker-buttons">
+  </div>
+  </div>
+
+</div>
+</template>
+
+<script>
+
+export default {
+  data: function () {
+    return {
+      containerPosition: {
+        top: "200px",
+        left: "250px",
+      },
+      oldColor: "",
+      itemID: "bo",
```

Comment:
This could be changed to tagName: ""

Reply:
Actually I'm thinking the code will be used for more then just tags -- ColorPicker could pick a color for a Style or GridElement. itemID could be the ID of any of them (which the ColorPicker.vue will know because of the "pickerType")

Reply:
Hmmm, thinking ahead huh... Cool!